scripts/codegen/CAN/App_CanMsgs.dbc

```diff
@@ -176,6 +176,9 @@ BO_ 505 DIM_REGEN_PADDLE: 8 DIM
 SG_ Raw_Paddle_Position : 0|32@1+ (1,0) [0|100] "%" LOGGER
 SG_ Mapped_Paddle_Position : 32|32@1+ (1,0) [0|100] "%" LOGGER
 
+BO_ 506 DIM_ROTARY_SWITCH: 1 DIM
+SG_ Drive_Mode : 0|3@1+ (1,0) [0|0] "" DCM
```

Comment:
Is the range supposed to be specified here?

Reply:
Yep fixed :+1: